### ISSUE TITLE
The link for a country not work

### ISSUE THREAD

**Issue by Mte90** [hallvors/mobilewebcompat#16]
The site actually not work, when you go on http://www.arewecompatibleyet.com/ the page is empty

**Comment by hallvors** [comment]
@Mte90 the UI was considered overly complex (it probably was.. :)) so I redesigned .. you now need to click a region and a country to see bug lists - or just use this link for Italy:
http://www.arewecompatibleyet.com/#list:it50

**Comment by Mte90** [comment]
The link not work, the page is empty, it's required a manual click on the country

**Comment by JeanPaulLucien** [comment]
The website doesn't work too. Cyberfox 42x64.

`unreachable code after return statement
 sitedata.js:3687:12
unreachable code after return statement
 sitedata.js:4585:75
unreachable code after return statement
 timelines.js:205:2
ReferenceError: masterBugTable is not defined
 timelines.js:27:1
"The version of Tabzilla you are using is deprecated and will be removed in the future.

Please update to the new static version here: https://github.com/mozilla/tabzilla/"
 tabzilla.js:612:17`

**Comment by hallvors** [comment]
masterBugTable not defined would be a rather bad problem for this site, but I can not reproduce that error..?

**Comment by Mte90** [comment]
I have the same errors but not for the `masterbugtable`. If I go in the home page, click on europe and italy i got:
```
TypeError: masterBugTable.lists[list].counts is undefined[Ulteriori informazioni]  timelines.js:424:57
```

**Comment by hallvors** [comment]
Hm.. now http://www.arewecompatibleyet.com/data/masterbugtable.js is indeed truncated. Perhaps the cron job keeping it updated gets killed by a timeout or something?

**Comment by JeanPaulLucien** [comment]
```
<div id="metrics">
<nav></nav>
<nav></nav>
</div>
<div id="details"></div>
```
Empty.

**Comment by hallvors** [comment]
All the data the site presents comes from the masterbugtable.js file. If this file is broken, the site is broken.
The file is created by running this script: https://github.com/hallvors/mobilewebcompat/blob/master/preproc/buildlists.py
This queries Bugzilla and the webcompat/web-bugs issues on Github to extract information on issues we want to track the status of.

It should run every two hours or so as a cron job. If it is broken temporarily, and then fixes itself it means sometimes the script is terminated before it writes the whole file or something. It would be great to modify it to write to a different file and check if the output is valid JS before moving the new file to overwrite masterbugtable.js.

**Comment by hallvors** [comment]
To fix linking directly to country lists, we need to make sure the hashchange function runs on load if location.hash has a value. This is the hashchange function: https://github.com/hallvors/mobilewebcompat/blob/master/js/timelines.js#L148

A crappy fix might be simply adding 
```
window.addEventListener('load', function(){
  if(location.hash){
    onhashchange({newURL: location.href});
  }
}, false);
```
Though it would be better to move most of the code in the existing `window.onhashchange` method to a method named handleHashValue or something, and call it from both onhashchange and onload..

**Comment by hallvors** [comment]
(Right now the site is broken again - #15 is important.. :( )

**Comment by Mte90** [comment]
My pr have the ode that load the section by the url

**Comment by hallvors** [comment]
Indeed, I merged it now. Hopefully @miketaylr can deploy an AWCY update.
